documentation/rosetta-modelling-component.md

```diff
@@ -1618,26 +1634,24 @@ Conditional mappings allow to build more complex mappings. Conditional mappings
 A set to mapping is used to set the value of an attribute to a constant value. They don\'t attempt to use any data from the input document as the value for the attribute and a synonym value must not be given. The type of the constant must be convertible to the type of the attribute. The constant value can be given as a string (converted as necessary) or an enum - e.g. :
 
 ```
-period PeriodEnum (1..1)
-  [synonym ISDA_Create_1_0 set to PeriodEnum.D]
-itemName string (1..1) <"In this ....">;
-  [synonym DTCC_11_0 set to "comment"]
+engineEnum EngineEnum (0..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_13 set to EngineEnum->HYBRID]
```

Comment:
same comment as above: do not use capital letters

Reply:
for hybrid

Reply:
ok, I'll change that